packages/trader/src/Modules/Trading/Containers/strike-param-modal.jsx

```diff
@@ -27,10 +27,8 @@ const StrikeParamModal = ({ is_open, toggleModal, strike, onChange, name, strike
                             i18n_default_text='If you buy a "<0>{{trade_type}}</0>" option, you receive a payout at expiry if the final price is {{payout_status}} the strike price. Otherwise, your “<0>{{trade_type}}</0>” option will expire worthless.'
                             components={[<strong key={0} />]}
                             values={{
-                                trade_type:
-                                    vanilla_trade_type === 'VANILLALONGCALL' ? localize('Call') : localize('Put'),
-                                payout_status:
-                                    vanilla_trade_type === 'VANILLALONGCALL' ? localize('above') : localize('below'),
+                                trade_type: localize(contract_subtype),
```

Comment:
Ohh, okay, I'll remove. 
I think any translation for words words `Call` and `Put` can be removed  because there is a confirmed decision from management that it should't be translated to any language except English.

Reply:
We should still wrap 'CALL' and 'PUT' in localize in case they have a change of heart. Which we can then pass it onto translation team. Less work for us in the future.